CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# 99.0.0
+
+In this major release we deleted and deprecated numerous APIs in ButtonBar, FeatureHighlight, List,
+Tabs, TextFields and Flexible Header. We fixed layout issue in Dialogs and BottomAppBar and added
+support for non-transient Snackbars.
```

Comment:
Please add a blurb about the new support for non-transient Snackbars. @yarneo for details.

Reply:
Done

Reply:
Please include this in a “New features” section and include a code snippet in either objective c or swift.

Reply:
Please include this in a “New features” section and include a code snippet in either objective c or swift.